src/main/resources/templates/db_scripts/updateItemStatusDate.sql

```diff
@@ -0,0 +1,23 @@
+
+-- Updates item status date if status name was changed.
+-- Item status date stores with timezone.
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.update_status_date() RETURNS TRIGGER
+AS $$
+  DECLARE
+	  newStatus text;
+  BEGIN
+  	newStatus = NEW.jsonb->'status'->>'name';
+  IF newStatus IS NULL THEN  RETURN NEW;
+  END IF;
+	IF (TG_OP='UPDATE') THEN
+	  IF (newStatus!=OLD.jsonb->'status'->>'name') THEN
+          NEW.jsonb = jsonb_set(NEW.jsonb, '{status,date}', to_jsonb(CURRENT_TIMESTAMP(3)), true);
```

Comment:
should we convert it to UTC time zone (+00:00)? I saw we use this time zone for metadata fields. 

@mreno-EBSCO what do you think?

Reply:
Hi @marcjohnson-kint, could you please advice which timezone should we use here? 

UTC with zero offset or DB server default (as in this case)?

Reply:
The timezone should be UTC